### PR TITLE
fix(blade): fix code & amount typography

### PR DESCRIPTION
**Pull request by anuraghazra**
Fixed minor bugs in Code & Amount components in ReactNative:  

Amount: 

- Missing lineHeight was causing humanize variant to cutoff from the top. 

|  Before | After  | 
|---|---|
|  image | image  | 

Code: 

- Fixed alignment
- Fixed width

|  Before | After  | 
|---|---|
|  image | image  | 

Before width fix: 

|  Before | After  | 
|---|---|
|  image | image
  |

### CODE REVIEW

File: packages/blade/package.json [hunk 1]
```diff
@@ -93,10 +93,10 @@
...
     "test:react": "FRAMEWORK=REACT jest -c ./jest.web.config.js",
     "test:react-native": "FRAMEWORK=REACT_NATIVE jest -c ./jest.native.config.js",
-    "start:ios": "run-p react-native:storybook:start react-native:storybook:ios",
-    "start:android": "run-p react-native:storybook:start react-native:storybook:android",
-    "start:native": "run-p react-native:storybook:start react-native:storybook:android react-native:storybook:ios",
-    "start:web": "yarn react",
+    "start:ios": "NODE_OPTIONS=--openssl-legacy-provider run-p react-native:storybook:start react-native:storybook:ios",
+    "start:android": "NODE_OPTIONS=--openssl-legacy-provider run-p react-native:storybook:start react-native:storybook:android",
+    "start:native": "NODE_OPTIONS=--openssl-legacy-provider run-p react-native:storybook:start react-native:storybook:android react-native:storybook:ios",
+    "start:web": "NODE_OPTIONS=--openssl-legacy-provider yarn react",
```
[the remark this answers is not in the document]
Comment (anuraghazra) [reply packages/blade/package.json:99]: These commands wont work if we don't add this. Courtesy of Node v18

**Review by divyanshu013 — DISMISSED** [review]
Looks good to me.

**Review by chaitanyadeorukhkar — APPROVED** [review]
Add changeset

**Review by chaitanyadeorukhkar — APPROVED** [review]
Add changeset

File: packages/blade/package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (saurabhdaware) [reply packages/blade/package.json:99]: Kamlesh already added them in master no 🤔

**Review by divyanshu013 — APPROVED** [review]
Looks good to me.

**Pull request merged**